tests/console/salt.pm

```diff
@@ -20,7 +20,7 @@ use utils qw(zypper_call pkcon_quit systemctl);
 sub run {
     select_console 'root-console';
     pkcon_quit;
-    zypper_call("in salt-master salt-minion", timeout => 180);
+    zypper_call('in salt-master salt-minion');
```

Comment:
Default timeout is 700, so it makes sense.

Reply:
of course. @Soulofdestiny did that in https://github.com/mnowaksuse/os-autoinst-distri-opensuse/commit/f12f84f555ad9c45c5882c4b4d38f9b7b8ecc59a#diff-9e5fa6e9dcc1ea49c0a44fa85f84b8ed and in hindsight that was … suboptimal ;)